src/ui/components/RatingsStatsPopover/index.tsx

```diff
@@ -6,6 +6,46 @@ import { helpers, toWorker } from "../../util";
 import ResponsivePopover from "../ResponsivePopover";
 import { PLAYER } from "../../../common";
 
+const PlayerNote = ({
+	note,
+	playerName,
+}: {
+	note: string | undefined;
+	playerName: string | undefined;
+}) => {
+	const [isOpen, setIsOpen] = useState(true);
+	if (!note) {
+		return null;
+	}
+	return (
+		<>
+			<div className="text-body-secondary bold">
+				{" "}
+				Notes {playerName !== undefined ? `for ${playerName}` : ""}
+			</div>
```

Comment:
I think this header can be deleted, people will know it's the note by the content, since you have to manually add a note for it to exist.